### PR TITLE
Rhel 9 add infra check

### PR DESCRIPTION
**Pull request by jkonecny12**
Backport of https://github.com/rhinstaller/anaconda/pull/3429 https://github.com/rhinstaller/anaconda/pull/3466 and https://github.com/rhinstaller/anaconda/pull/3467.

~[Test run](https://github.com/jkonecny12/anaconda/runs/2885238403?check_suite_focus=true).~ New [test run](https://github.com/jkonecny12/anaconda/pull/13/checks?check_run_id=2896380258).

### CODE REVIEW

**Comment by jkonecny12** [comment]
@jstodola asking for exception.

**Review by VladimirSlavik — APPROVED** [review]
Thank you!

**Review by poncovka — APPROVED** [review]

**Comment by jkonecny12** [comment]
This PR have to be extended by https://github.com/rhinstaller/anaconda/pull/3466 .

File: .structure-config [hunk 1]
```diff
@@ -0,0 +1 @@
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```
Comment (jstodola) [inline .structure-config:1]: `.coveragerc` is part of anaconda-34.25.0.9-1.el9.src.rpm - is it expected? If yes, then is should not be marked as a infrastructure file.

Comment (jkonecny12) [reply .structure-config:1]: The same answer as in https://github.com/rhinstaller/anaconda/pull/3462#discussion_r660553140 .

**Comment by jkonecny12** [comment]
**UPDATED:**
* coveragerc was removed from the tarball

**Review by VladimirSlavik — APPROVED** [review]

**Comment by jkonecny12** [comment]
Rebased.

**Review by jstodola — APPROVED** [review]
It looks good to me, thanks. (sorry for the delay)

**Pull request merged**